make/autoconf/lib-bundled.m4

```diff
@@ -23,6 +23,12 @@
 # questions.
 #
 
+#
+# ===========================================================================
+# (c) Copyright IBM Corp. 2021, 2021 All Rights Reserved
+# ===========================================================================
+#
```

Comment:
not: Please remove lines 26 & 30 for consistency with other files modified by openj9.

Reply:
Resolved d3055c6